main.py

```diff
@@ -1,10 +1,51 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = json.load(file)
+
+    for player, data in players_data.items():
+        race_data = data["race"]
+        if not Race.objects.filter(name=race_data["name"]).exists():
+            Race.objects.create(
```

Comment:
If you assign `create()` put output to the `race` value here and put the `get()` into an `else` block, it will reduce amount of database calls at the cost of one line of code. I think it's worth it.
I'd suggest to apply it everywhere in this file, not only to race creation.